# Sensor TF publish rate
sensor_tf_rate: 10

# AABB tree parameters
distance_threshold: -1.0    # Distance above which points do not contribute to
                            # the minimization problem. Negative distance 
                            # indicates infinite threshold, i.e. all points will
                            # contribute. Small value might help with outliers

# CICP parameters
nn_normals: 10      # How many neighbors to consider for normal computation
max_features: 3     # Max features to retain per voxel
min_elements: 1     # Minimum points required for a feature to be valid
voxel_size: 0.1     # Size of voxel in world units (for e.g. meters)
merge_eps: 0.25     # Norm difference below which normals may be merged

# R-LOAM parameters
min_cardinality: 100  # Minimum cardinality of geometric features. This allows 
                      # to not give too much importance in case of low geometric
                      # features
lambda: 1.0           # The lambda term in R-LOAM. If set to 1, corner, flat and
                      # geometric points have equal weights. If set to 2, 
                      # geometric points have equal weight to LOAM points

# Visualization
visualize: false    # To visualize or not to visualize
scale: 0.05         # Scale of normals

# CICP node
parallel_solver: true       # Run optimizer in parallel mode
print_optimization: false   # Set to true to print ceres optimized pose
